examples/existing-image/terraform.tfvars.example

```diff
@@ -0,0 +1,8 @@
+acm_certificate_arn=""
```

Comment:
Can you please make this example with dummy values, like [this one](https://github.com/hashicorp/terraform-azurerm-terraform-enterprise/blob/main/examples/standalone_airgap/terraform.tfvars.example)?